plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -105,15 +105,28 @@ const fetchPluginData = async ({createNode, reporter}) => {
     const pipelinePluginsUrl = 'https://www.jenkins.io/doc/pipeline/steps/contents.json';
     const pipelinePluginIds = await requestGET({url: pipelinePluginsUrl, reporter});
     const bomDependencies = await fetchBomDependencies(reporter);
+    const updateUrl = 'https://updates.jenkins.io/update-center.actual.json';
+    const updateData = await requestGET({url: updateUrl, reporter});
     do {
         const url = `https://plugins.jenkins.io/api/plugins/?limit=100&page=${page}`;
         pluginsContainer = await requestGET({url, reporter});
         for (const plugin of pluginsContainer.plugins) {
             const pluginName = plugin.name.trim();
             names.push(pluginName);
+            const pluginUC = updateData.plugins[pluginName];
+            pluginUC.maintainers = pluginUC.developers;
+            pluginUC.maintainers.forEach(maint => {maint.id = maint.developerId, delete(maint.developerId);});
```

Comment:
lets just update the few uses of "maintainers" to be "developers" instead of messing with the data too much?